modules/rank-eval/src/main/java/org/elasticsearch/index/rankeval/DiscountedCumulativeGainAt.java

```diff
@@ -84,6 +86,19 @@ public DiscountedCumulativeGainAt(int position) {
     }
 
     /**
+     * @param position number of top results to check against a given set of relevant results. Must be positive.
+     *  // TODO is there a way to enforce this?
```

Comment:
We check this using the existing constructor that only takes the position argument.